spectrum-scanner/Makefile

```diff
@@ -9,9 +9,10 @@ RIOTBASE ?= $(CURDIR)/../RIOT
 
 # Define modules that are used
 USEMODULE += gnrc
-USEMODULE += gnrc_netdev_default
+USEMODULE += netdev_default
 USEMODULE += auto_init_gnrc_netif
 USEMODULE += xtimer
+USEMODULE += ztimer64_xtimer_compat
```

Comment:
Why not use `ztimer64` outright?

Reply:
Works just as well, now using that.

Reply:
Mhhh I accidentally commented on the wrong file, but you fixed it for the correct one ^^, but I guess the same question is true for `spectrum-scanner` too.